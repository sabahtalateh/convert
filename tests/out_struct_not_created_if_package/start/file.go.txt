package start

type In struct {}

//go:generate convert
func Convert(in *In) *a.Out {}
