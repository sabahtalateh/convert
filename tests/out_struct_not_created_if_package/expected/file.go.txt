package start

type In struct {}

type Out struct {
	A string
}

func Convert(in *In) *Out {
	var out *Out

	// no field: In.A

	return out
}
